src/app/components/ATIAnalytics/canonical/sendBeaconOperaMiniScript.ts

```diff
@@ -0,0 +1,12 @@
+import isOperaProxy from '#app/lib/utilities/isOperaProxy';
+
+const sendBeaconOperaMiniScript = (atiPageViewUrlString: string) => `
+if ((${isOperaProxy.toString()})()) {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', '${atiPageViewUrlString}', true);
```

Comment:
There may need to be some string escaping judging by the tests, as `atiPageViewUrlString` could have `'` within them.

Reply:
Yeah there's also a load of code coverage stuff breaking in tests, trying to figure how to fix it, doing `/* istanbul ignore next */` doesn't seem to work https://jestjs.io/docs/configuration#collectcoverage-boolean

Reply:
I've remedied this by using double quotes instead of single quotes. And I've swapped const for var for compatibility (ran it through babel)